database/aws-s3.js

```diff
@@ -5,13 +5,34 @@ const AWS = require('aws-sdk');
 
 const s3v2 = new AWS.S3();
 
-const s3 = new S3Client({ region: process.env.AWS_REGION });
+const s3 = new S3Client({ Region: process.env.AWS_REGION });
 
 AWS.config.update({
   accessKeyId: AWS_ACCESS_KEY_ID,
   secretAccessKey: AWS_SECRET_ACCESS_KEY
 });
 
+//list photos from s3 bucket
+const listPhotos = async () => {
+
+  const getParams = {
+    Bucket: 'sdc-airbnb-photos',
+  }
+
+  return new Promise(async (resolve, reject) => {
+    try {
+      const bucket = await s3v2.listObjects(getParams, (err, data) => {
+        if (err) console.log(err);
+        else {
+          resolve(data)
+        }
+      });
+    } catch(err) {
+      console.log(err);
+    }
+  })
+}
+
 // Get all buckets
 const getBuckets = async () => {

```

Comment:
just out of curiosity, when would this function be used?

Reply:
the `getBuckets` function is leftover from FEC, I don't have use for it.. will clean up this file soon.